lib/logstash/inputs/s3.rb

```diff
@@ -138,11 +138,11 @@ def list_new_files
         if ignore_filename?(log.key)
           @logger.debug('S3 input: Ignoring', :key => log.key)
         elsif log.content_length <= 0
-          @logger.debug('S3 Input: Object Zero Length', :key => log.key)
+          @logger.debug('S3 input: Object Zero Length', :key => log.key)
```

Comment:
Plugins will log their class already, due to how log4j is configured. maybe we can just do away with these "S3 input" prefixes? I see some log lines in this plugin that don't have it.

Reply:
yeah, was thinking that as well. 
the ones that deal with "remote" S3 objects have the prefix.
while the others, when the file is downloaded and being processed locally, do not prefix.